makefile

```diff
@@ -1,3 +1,7 @@
+ifndef OS
+OS:=$(shell uname)
+endif
```

Comment:
I've done a little more research.  If you accept the claim in https://stackoverflow.com/a/12099167 that `OS` will be set in the environment on Windows systems, this should work fine for Windows in addition to systems with `uname`.